packages/client/src/i18n/english.ts

```diff
@@ -82,6 +82,8 @@ export type ENGLISH =
   | readonly [PORTFOLIO_CHART_TYPES]
   | readonly ["Total"]
   | readonly ["Available"]
+  | readonly ["Delegated"]
```

Comment:
Nice 👍 

It looks like you looked into and figured out the internationalization scheme?

Looks like you got it! But just a quick note for reference:

* The source of truth is this `english.ts` file, which basically provides type definitions for all internationalized text. This helps to ensure all rendered text has a translation.
* The translations are all in `.json` files in the `i18n/catalogs/` folder.
* The command `yarn i18n` will update the `.json` files to match the `english.ts` source file, adding the placeholder `"<<< TODO: Implement >>>"` strings for new keys without translations. These can then later be manually translated.
* The scheme should fallback to the english text, if the translation has not been implemented yet.
* The type definitions ensure that the JSON files contain only keys which exist in the `english.ts` source file, and the translation methods `t` and `tString` also ensure the provided inputs are correct.
* This approach also supports interpolating dynamic values, and type checking for this as well.

More details can be seen here: https://github.com/ChorusOne/anthem/tree/master/packages/client/src/i18n.